content/java/concepts/strings/terms/hashCode/hashCode.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.hashCode()'
+Description: 'Returns an integer hash code value for the object on which it is invoked.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`hashCode()`** method returns an integer hash code value for the object on which it is invoked. The hash code for a String object is computed like this:
+
+```pseudo
+s[0]*31^(n-1) + s[1]*31^(n-2) + ... + s[n-1]
+```
+
+## Syntax
+
+```pseudo
+int hashCodeValue = objectName.hashCode();
+```
+
+- `hashCodeValue` is an integer variable that stores the hash code value returned by the .hashCode() method.
+- `objectName` is the object for which we want to obtain the hash code value.
```

Comment:
```suggestion
- `hashCodeValue` is an integer variable that stores the hash code value returned by the `.hashCode()` method.
- `objectName` is the object for which we want to obtain the hash code value.
```

Let's refrain from using pronouns like "we" and just the second bullet point accordingly